data-solutions/data-platform-foundations/01-environment/main.tf

```diff
@@ -38,6 +38,9 @@ module "project-datamart" {
     bq      = [var.service_encryption_key_ids.multiregional]
     storage = [var.service_encryption_key_ids.multiregional]
   }
+  # If used, remember to uncomment 'lifecycle' block in the
```

Comment:
Is there a better way of doing this? At a minimum, it should be clearly detailed in the README.

Reply:
At the moment the default is to handle parimeter<->project mapping in the vpc-sc module. At the moment I do not see a better way to address this, added a comment in the README. In the future we may change the default behavior and set the default in the other way around.